### DISCUSSION TITLE
Failed to register: StatusCode: 429 (RegistrationRetryException) - signal-cli 0.12.2

### DISCUSSION THREAD

**Discussion by tootai** [AsamK/signal-cli#1343] [General]
Hi

since yesterday I try to register a landline with 
signal-cli -v -u +33345678901 register --voice --captcha signal-hcaptchaRestOfDatas

and always get subject message. I generated few captchas, doesn't change anything. What am I doing wrong ?

Daniel

**Comment by AsamK** [comment]
you probably need to try to register with sms verification first, see https://github.com/AsamK/signal-cli/issues/1202

**Reply by tootai** [reply]
Thanks ! Ce process described by csouti-chaps did the job.

**Comment by tootai** [comment]
Solved
